__tests__/app.test.js

```diff
@@ -105,6 +105,39 @@ describe("CORE:",() => {
                 })
             })
         })
+        describe("PATCH:", () => {
+            it("200: patches given article", () => {
+                return request(app)
+                .patch("/api/articles/1")
+                .send({
+                    inc_votes : 1 
+                })
+                .expect(200)
+                .then((response) => {
+                    const {
+                        body: { update },
+                    } = response;
+                    expect(update).toHaveLength(1)
```

Comment:
Shouldn't need to check the length as you shouldn't need to send an array back, return rows[0] in your model to just get the object back